felix/fv/infrastructure/infra_k8s.go

```diff
@@ -210,8 +210,14 @@ func RunBPFLog() *containers.Container {
 }
 
 func (kds *K8sDatastoreInfra) runK8sApiserver() {
+	// Get current working dir as docker does not accept relative paths for mounting volumes
+	pwd, err := os.Getwd()
+	if err != nil {
+		panic(err)
+	}
 	args := []string{
 		"-v", os.Getenv("CERTS_PATH") + ":/home/user/certs", // Mount in location of certificates.
+		"-v", pwd + "/../../libcalico-go/config/crd:/crds", // Mount in location of CRDs.
```

Comment:
I'm not entirely sure but I was having failures almost every time when trying to copy the CRDs into the container (maybe related to low disk space on my machine), mounting as volume fixed it